src/main/webapp/js/instructor.es6

```diff
@@ -369,6 +369,25 @@ function attachEventToDeleteStudentLink() {
     });
 }
 
+function remindButtonAjax(urlLink) {
```

Comment:
@chowyb which would be better `remindStudents` , `sendRemindersToStudents` , `sendReminders` , `generateRemindButtonResponse` ?